package.json

```diff
@@ -75,7 +75,20 @@
     "unist-util-visit-parents": "^2.1.2"
   },
   "devDependencies": {
+    "@babel/core": "^7.19.6",
+    "@chakra-ui/storybook-addon": "^4.0.12",
     "@netlify/functions": "^1.2.0",
+    "@storybook/addon-a11y": "^6.5.13",
+    "@storybook/addon-actions": "^6.5.13",
+    "@storybook/addon-essentials": "^6.5.13",
+    "@storybook/addon-interactions": "^6.5.13",
+    "@storybook/addon-links": "^6.5.13",
+    "@storybook/builder-webpack4": "^6.5.13",
```

Comment:
webpack4 related packages shouldn't be necessary, right?

Reply:
Good point. This was generated automatically on init. Weird that is did that along with the webpack5.

Also, when reviewing with a fresh Gatsby project and checking out build processes when publishing storybook, `webpack` might be needed as a devDep due to a [webpack module error](https://github.com/storybookjs/storybook/issues/15336)